en/attributes.html

```diff
@@ -186,22 +186,28 @@ <h2 id="index-structures">Index structures</h2>
 </p>
 <p>
   Setting <a href="reference/schema-reference.html#attribute">fast-search</a>
-  creates a dictionary, an index structure for quicker lookup.
+  creates a <a href="reference/schema-reference.html#dictionary">dictionary</a>,
+  an index structure for quicker lookup.
   This uses more memory, and also more CPU when updating documents.
   It increases steady state memory usage for all attribute types
   and also add initialization overhead for numeric types.
-  The default index structure is a b-tree of attribute <em>values</em>,
-  pointing to an <em>occurrence</em> b-tree of local doc IDs for each value:
 </p>
-<img src="img/attributes-indexes.svg" width="385" height="auto" />
+<p>
+  The default dictionary is a b-tree of attribute <em>values</em>,
+  pointing to an <em>occurrence</em> b-tree of local doc IDs for each value, exemplified in the A-attribute below.
+  Using <code>dictionary: hash</code> on the attribute generates a hash table of with posting list values,
+  as in the C-attribute:
```

Comment:
Not sure how the illustration shows the posting lists, but we might need to mention here that "short posting lists are represented as arrays instead of b-trees".